systemtests/util_test.go

```diff
@@ -158,9 +163,25 @@ func (s *systemtestSuite) rebootstrap() error {
 	return nil
 }
 
+func getDriver() string {
+	driver := "ceph"
+	if strings.Trim(os.Getenv("USE_DRIVER"), " ") != "" {
+		driver = strings.Trim(os.Getenv("USE_DRIVER"), " ")
+	}
+	return driver
+}
+
+func getDriverDir() string {
+	driverDir := ""
+	if getDriver() != "ceph" {
+		driverDir = getDriver()
+	}
+	return driverDir
+}
+
```

Comment:
we should not do this; we should provde the `testdata` dirs for ceph too, and avoid this guard and extra code.